image/verify/src/verifier.rs

```diff
@@ -438,6 +438,18 @@ impl<Env: ImageVerificationEnv> ImageVerifier<Env> {
             Err(CaliptraError::IMAGE_VERIFIER_ERR_FMC_RUNTIME_INCORRECT_ORDER)?;
         }
 
+        // Check if fmc and runtime images don't overlap on loading in ICCM.
+        let fmc_load_addr_start = manifest.fmc.load_addr;
```

Comment:
we should check if the load adress is located in iccm or is it being done already?

Reply:
It's being done already.